### DISCUSSION TITLE
Wrong results with "large" modal

### DISCUSSION THREAD

**Discussion by tcl8273** [openai/whisper#590] [Q&A]
I ran whisper with the base modal and got the correct result, but the large modal is wrong.
BASE MODAL: "{'text': " to get married and for the rest of the life, there'll be in the style to which to become accustomed. Well, it is really, really difficult. And I think it comes down to...", 'segments': [{'id': 0, 'seek': 0, 'start': 0.0, 'end': 2.56, 'text': ' to get married and for the rest of the life,', 'tokens': [50364, 281, 483, 5259, 293, 337, 264, 1472, 295, 264, 993, 11, 50492, 50492, 456, 603, 312, 294, 264, 3758, 281, 597, 281, 1813, 35980, 13, 50654, 50654, 1042, 11, 309, 307, 534, 11, 534, 2252, 13, 50766, 50766, 400, 286, 519, 309, 1487, 760, 281, 485, 50866], 'temperature': 0.0, 'avg_logprob': -0.3613230646873007, 'compression_ratio': 1.328125, 'no_speech_prob': 0.07791075855493546}, {'id': 1, 'seek': 0, 'start': 2.56, 'end': 5.8, 'text': " there'll be in the style to which to become accustomed.", 'tokens': [50364, 281, 483, 5259, 293, 337, 264, 1472, 295, 264, 993, 11, 50492, 50492, 456, 603, 312, 294, 264, 3758, 281, 597, 281, 1813, 35980, 13, 50654, 50654, 1042, 11, 309, 307, 534, 11, 534, 2252, 13, 50766, 50766, 400, 286, 519, 309, 1487, 760, 281, 485, 50866], 'temperature': 0.0, 'avg_logprob': -0.3613230646873007, 'compression_ratio': 1.328125, 'no_speech_prob': 0.07791075855493546}, {'id': 2, 'seek': 0, 'start': 5.8, 'end': 8.040000000000001, 'text': ' Well, it is really, really difficult.', 'tokens': [50364, 281, 483, 5259, 293, 337, 264, 1472, 295, 264, 993, 11, 50492, 50492, 456, 603, 312, 294, 264, 3758, 281, 597, 281, 1813, 35980, 13, 50654, 50654, 1042, 11, 309, 307, 534, 11, 534, 2252, 13, 50766, 50766, 400, 286, 519, 309, 1487, 760, 281, 485, 50866], 'temperature': 0.0, 'avg_logprob': -0.3613230646873007, 'compression_ratio': 1.328125, 'no_speech_prob': 0.07791075855493546}, {'id': 3, 'seek': 804, 'start': 8.04, 'end': 33.04, 'text': ' And I think it comes down to...', 'tokens': [50364, 400, 286, 519, 309, 1487, 760, 281, 485, 51614], 'temperature': 0.0, 'avg_logprob': -0.5287182547829368, 'compression_ratio': 0.7948717948717948, 'no_speech_prob': 0.026778418570756912}], 'language': 'en'}"

LARGE MODAL: "{'text': " and for the rest of their life they'll be in the style to which they've become accustomed.", 'segments': [{'id': 0, 'seek': 0, 'start': 0.0, 'end': 7.0, 'text': " and for the rest of their life they'll be in the style to which they've become accustomed.", 'tokens': [50389, 293, 337, 264, 1472, 295, 641, 993, 436, 603, 312, 294, 264, 3758, 281, 597, 436, 600, 1813, 35980, 13, 50714], 'temperature': 0.0, 'avg_logprob': -0.2422804832458496, 'compression_ratio': 1.1538461538461537, 'no_speech_prob': 0.1417381912469864}], 'language': 'en'}"

Can you help me check this error? And I need LARGE MODAL returns like BASE MODAL
Thank you so much!

**Comment by bmilde** [comment]
Might be related, I had problems with the transcribe() function giving different results vs. using the Whisper commandline.  See https://github.com/openai/whisper/discussions/591
